File: soutLink-frontend/src/app/services/models/student-appointment-response.ts
/* tslint:disable */
/* eslint-disable */
import { StudentResponse } from '../models/student-response';
export interface StudentAppointmentResponse {
  appointmentDate?: string;
  appointmentTime?: string;
  createdDate?: string;
  description?: string;
  id?: number;
  objet?: string;
  status?: 'WAITING' | 'PENDING' | 'ACCEPTED' | 'CANCELED';
  student?: StudentResponse;
}

File: soutLink-backend/src/main/java/com/soutLink/supervisor/dto/StudentAppointmentResponse.java
package com.soutLink.supervisor.dto;

import com.soutLink.enums.AppointmentStatus;
import lombok.*;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class StudentAppointmentResponse {
    private Long id;
    private String description;
    private AppointmentStatus status;
    private String objet;
    private StudentResponse student;
    private String createdDate;
    private String appointmentDate;
    private String appointmentTime;
}
